src/fields/email.ts

```diff
@@ -8,7 +8,13 @@ export interface IEmailSchema<V = undefined> extends IFieldSchemaBase<"email", V
 export const email: IFieldGenerator<IEmailSchema> = (id, { validation }) => {
 	const emailRule = validation?.find((rule) => rule.email);
 
+	let schema = Yup.string();
+	// no need to apply if it is defined as it will be applied in the mapRules() function
+	if (!emailRule?.email) {
```

Comment:
very nitpicky but `!emailRule` is sufficient because it was already filtered in the find predicate

Reply:
yeah it's just to be extra safe in case there is a `{ "email": false }` rule

Reply:
good catch, I have made the update